docs/streaming/ss-migration-guide.md

```diff
@@ -23,6 +23,10 @@ Note that this migration guide describes the items specific to Structured Stream
 Many items of SQL migration can be applied when migrating Structured Streaming to higher versions.
 Please refer [Migration Guide: SQL, Datasets and DataFrame](../sql-migration-guide.html).
 
+## Upgrading from Structured Streaming 4.0 to 4.1
+
+- Since Spark 4.1, AQE is supported for stateless workloads, and it could affect the behavior of the query after upgrade (especially since AQE is turned on by default). In general, it helps to achieve better performance including resolution of skewed partition, but you can turn off AQE via changing `spark.sql.adaptive.enabled` to `false` to restore the behavior if you see regression.
```

Comment:
shall we have a dedicated config for streaming AQE? It seems overkill to turn off AQE entirely.

Reply:
I'm OK with introducing a config to control over streaming AQE - should we do this as layered configs? e.g. If AQE is turned off via spark.sql.adaptive.enabled, we turn off both batch and streaming. The config for streaming AQE takes effect only when the AQE config spark.sql.adaptive.enabled is turned on.

Reply:
layered config SGTM.